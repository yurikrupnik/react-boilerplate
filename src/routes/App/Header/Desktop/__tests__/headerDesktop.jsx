import React from 'react';
import { StaticRouter } from 'react-router-dom';
import Component from '../index';

const {
    it,
    expect,
    mount
} = global;

it('renders <Component /> component', () => {
    const wrapper = mount((
        <StaticRouter context={{}}>
            <Component />
        </StaticRouter>
    ));
    expect(wrapper).toMatchSnapshot();
});
