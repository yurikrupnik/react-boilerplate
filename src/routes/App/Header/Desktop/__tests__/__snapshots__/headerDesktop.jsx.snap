// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders <Component /> component 1`] = `
<StaticRouter
  basename=""
  context={Object {}}
  location="/"
>
  <Router
    history={
      Object {
        "action": "POP",
        "block": [Function],
        "createHref": [Function],
        "go": [Function],
        "goBack": [Function],
        "goForward": [Function],
        "listen": [Function],
        "location": Object {
          "hash": "",
          "pathname": "/",
          "search": "",
          "state": undefined,
        },
        "push": [Function],
        "replace": [Function],
      }
    }
  >
    <DesktopNav>
      <div>
        <CurrencyConsumer
          render={[Function]}
        >
          <DataContainer
            data={Array []}
            fetch={[Function]}
            loading={false}
            render={[Function]}
            selected={null}
          >
            <div>
              header deskotp
              <div />
              <div>
                <Link
                  replace={false}
                  to="/"
                >
                  <a
                    href="/"
                    onClick={[Function]}
                  >
                    Main
                  </a>
                </Link>
              </div>
              <div
                className="row"
              >
                <div
                  className="col-xs"
                  key="guarantee"
                >
                  <Link
                    replace={false}
                    to="/guarantee"
                  >
                    <a
                      href="/guarantee"
                      onClick={[Function]}
                    >
                      guarantee
                    </a>
                  </Link>
                </div>
                <div
                  className="col-xs"
                  key="topics"
                >
                  <Link
                    replace={false}
                    to="/topics"
                  >
                    <a
                      href="/topics"
                      onClick={[Function]}
                    >
                      topics
                    </a>
                  </Link>
                </div>
                <div
                  className="col-xs"
                  key="education"
                >
                  <Link
                    replace={false}
                    to="/education"
                  >
                    <a
                      href="/education"
                      onClick={[Function]}
                    >
                      education
                    </a>
                  </Link>
                </div>
              </div>
            </div>
          </DataContainer>
        </CurrencyConsumer>
      </div>
    </DesktopNav>
  </Router>
</StaticRouter>
`;
